### PR TITLE
Css for OpenCity, MonsInsurance, MolecularPrinting, NuclearZone, Midas

### PR DESCRIPTION
**Pull request by nwai90**
Screenshot 2020-10-13 at 11 17 43 AM
Screenshot 2020-10-13 at 11 17 37 AM
Screenshot 2020-10-13 at 11 17 27 AM
Screenshot 2020-10-13 at 11 17 10 AM
Screenshot 2020-10-13 at 11 17 15 AM

### CODE REVIEW

**Comment by nwai90** [comment]
@Lynesth I'll check French version of card later

**Comment by nwai90** [comment]
@Lynesth This should be fine now

Screenshot 2020-10-13 at 1 47 14 PM

File: src/HTML_data.ts [hunk 1]
```diff
@@ -1194,11 +1194,10 @@ export const HTML_DATA: Map<string, string> =
                 <div class="production-prefix minus"></div><div class="energy production "></div>
                 <div class="production-prefix plus"></div><div class="money production ">4</div>
             </div>
-            <div class="tile city-tile " style="margin-left:20px;margin-bottom:5px; "></div><br>
+            <div class="tile city-tile " style="margin-left:20px;margin-bottom:5px; "></div>
             <div class="plant resource "></div><div class="plant resource "></div>
-            <div class="description " style="margin-top:-8px;font-size:11px;text-align: left">
-                <div>Requires 12% oxygen. Gain 2 plants. Place a City tile.</div>
-                <div style="width: 135px">Decrease your Energy production 1 step and increase your MC production 4 steps.</div>
+            <div class="description " style="margin-top:15px;font-size:11px;text-align: left">
```
Comment (alrusdi) [inline src/HTML_data.ts]: No inline styles please

Comment (nwai90) [reply src/HTML_data.ts]: 👌  Removed

**Review by alrusdi — APPROVED** [review]
Thank you! It is great

**Pull request merged**